package.json

```diff
@@ -11,14 +11,50 @@
     "url": "git://github.com/conveyal/otp.js.git"
   },
   "devDependencies": {
-    "component": "*",
-    "component-uglifyjs": "*",
+    "babel-plugin-syntax-dynamic-import": "^6.18.0",
+    "backbone": "^1.2.1",
+    "bootstrap": "^3.3.5",
+    "eonasdan-bootstrap-datetimepicker": "^3.1.3",
+    "css-loader": "^2.1.1",
+    "es5-shim": "^4.5.13",
+    "es6-shim": "^0.35.0",
+    "file-loader": "^3.0.1",
+    "handlebars": "^4.1.2",
+    "haversine": "^1.1.1",
+    "html-loader": "^0.5.5",
+    "inquirer": "^6.3.1",
+    "jquery": "^1.11.3",
+    "leaflet": "^0.7.3",
+    "loadash": "^1.0.0",
+    "moment": "^2.10.3",
+    "raphael": "^2.1.4",
+    "select2": "^3.5.1",
     "serve": "^1.4.0",
-    "standard": "8.3.0"
+    "standard": "^8.3.0",
+    "style-loader": "^0.23.1",
+    "uglifyjs-webpack-plugin": "^2.1.2",
+    "underscore": "^1.8.3",
+    "webpack": "^4.31.0",
+    "webpack-cli": "^3.3.2"
   },
   "scripts": {
     "start": "serve client",
+    "watch": "webpack --watch",
+    "webpack": "webpack",
```

Comment:
The more intuitive name for this `webpack` script to me is `build`. It keeps the command build tool-independent and is more of a conventional name.